controls/radrichtextbox/features/selection.md

```diff
@@ -10,32 +10,112 @@ published: True
+The user is able to select the content inside **RadRichTextBox** in the same way as in MS Word. This is done by clicking on the desired position and dragging to the desired end of the selection. A multiple ranges selection is also allowed. This one is done by holding the **Ctrl** key while selecting the different ranges.
+
+You can modify the appearance of the selection in the control through the **SelectionFill** and **SelectionStroke** properties of **RadRichTextBox**.
+
+
+#### [XAML] Example 1: Customize the appearance of the selection
+{{region radrichtextbox-features-selection_8}}
+
+	<telerik:RadRichTextBox Name="radRichTextBox" SelectionStroke="DarkRed">
+	    <telerik:RadRichTextBox.SelectionFill>
+	        <SolidColorBrush Color="Red" Opacity="0.5" />
+	    </telerik:RadRichTextBox.SelectionFill>
+	</telerik:RadRichTextBox>
+{{endregion}}
+
+
+#### [C#] Example 1: Customize the appearance of the selection
+{{region radrichtextbox-features-selection_6}}
+
+	this.radRichTextBox.SelectionFill = new System.Windows.Media.SolidColorBrush(Colors.Red); 
+	this.radRichTextBox.SelectionFill.Opacity = 0.2;
+	this.radRichTextBox.SelectionStroke = System.Windows.Media.Brushes.DarkRed;
+{{endregion}}
+
+
+#### [VB.NET] Example 1: Customize the appearance of the selection
+{{region radrichtextbox-features-selection_7}}
+	
+	Me.radRichTextBox.SelectionFill = New SolidColorBrush(Colors.Red)
+	Ме.radRichTextBox.SelectionFill.Opacity = 0.2
+	Ме.radRichTextBox.SelectionStroke = Brushes.DarkRed
+{{endregion}}
+
+#### Figure 1: Custom selection colors
+![Customized Selection in RadRichTextBox](images/RadRichTextBox_Selection_02.png)
+
 
 ## Programmatic Selection
 
-The developer is allowed to work with the selection programmatically. This can be used when implementing [Search functionality]({%slug radrichtextbox-features-search%}) functionality in your __RadRichTextBox__ and you want to select the found string. 
+The developer is allowed to manipulate the selection programmatically. This can be used in a lot of scenarios involving editing of the document, or for example, when implementing [Search functionality]({%slug radrichtextbox-features-search%}) in your **RadRichTextBox** and you want to select the found string. 
+
+The programmatic selection is represented via the **DocumentSelection** class. The instance of the class gets associated with the **RadDocument** of **RadRichTextBox** and allows you to specify selection start and end positions, selection ranges and other. You can manage the selection by either using the **Selection** property of **RadDocument** or by creating an instance of the **DocumentSelection** class.
+
+>tipTo learn more about the **DocumentPosition** read the [Positioning]({%slug radrichtextbox-features-positioning%}) topic.
+
+
+### Members of DocumentSelection

```

Comment:
Me personally don't like the sections that lists the public methods and properties of the class. This information is easily accessible in the API Reference. Or by clicking F12 on the class. 

I think the better approach here would be to show the methods using code examples and images where applicable. For example, the SelectTableRow, SelectTableColumn and AddTableCellToSelection methods could be combined into a section called something like "Select Table Elements". And here you can include 3 code snippets showing how to select rows, columns, and cells. Maybe you can add also picutres.

Anyway, this feedback is optional so you don't need to make changes here.